sway-lib-std/src/inputs.sw

```diff
@@ -13,112 +17,273 @@ use ::tx::{
     Transaction,
     tx_type,
 };
+use core::ops::Eq;
 
 const GTF_INPUT_TYPE = 0x101;
 
 ////////////////////////////////////////
 // GTF Opcode const selectors
 ////////////////////////////////////////
-// const GTF_INPUT_COIN_TX_ID = 0x102;
-// const GTF_INPUT_COIN_OUTPUT_INDEX = 0x103;
-const GTF_INPUT_COIN_OWNER = 0x104;
-// const GTF_INPUT_COIN_AMOUNT = 0x105;
-// const GTF_INPUT_COIN_ASSET_ID = 0x106;
-// const GTF_INPUT_COIN_TX_POINTER = 0x107;
-// const GTF_INPUT_COIN_WITNESS_INDEX = 0x108;
-// const GTF_INPUT_COIN_MATURITY = 0x109;
-// const GTF_INPUT_COIN_PREDICATE_LENGTH = 0x10A;
-// const GTF_INPUT_COIN_PREDICATE_DATA_LENGTH = 0x10B;
-// const GTF_INPUT_COIN_PREDICATE = 0x10C;
-const GTF_INPUT_COIN_PREDICATE_DATA = 0x10D;
-
-// const GTF_INPUT_CONTRACT_TX_ID = 0x10E;
-// const GTF_INPUT_CONTRACT_OUTPUT_INDEX = 0x10F;
-// const GTF_INPUT_CONTRACT_BALANCE_ROOT = 0x110;
-// const GTF_INPUT_CONTRACT_STATE_ROOT = 0x111;
-// const GTF_INPUT_CONTRACT_TX_POINTER = 0x112;
-// const GTF_INPUT_CONTRACT_CONTRACT_ID = 0x113;
-// const GTF_INPUT_MESSAGE_MESSAGE_ID = 0x114;
-// const GTF_INPUT_MESSAGE_SENDER = 0x115;
-const GTF_INPUT_MESSAGE_RECIPIENT = 0x116;
-
-// const GTF_INPUT_MESSAGE_AMOUNT = 0x117;
-// const GTF_INPUT_MESSAGE_NONCE = 0x118;
+// pub const GTF_INPUT_COIN_TX_ID = 0x102;
```

Comment:
Should these be commented out?

Reply:
For now, yes. They are being enabled by consecutive PRs and help to highlight what is still needed.